botorch/acquisition/analytic.py

```diff
@@ -231,6 +234,70 @@ def forward(self, X: Tensor) -> Tensor:
         return Phi(u)
 
 
+class ApproximateProbabilityOfImprovement(AnalyticAcquisitionFunction):
```

Comment:
Feels like this should have a notion of batch support in the name? Maybe sth like `qAnalyticProbabilityOfImprovement` ? Or `BatchProbabilityOfImprovement`?